open-policy-agent/trusted-image-sources/disallowed-registries/README.md

```diff
@@ -0,0 +1,9 @@
+# Disallow Registries
+
+The policy makes sure a list of "Allowed Registries" is associated with the `cluster` Image resource. If a registry is not mentioned in the Image resource, images from this registry will not be pulled for pod creation.
```

Comment:
What what do you mean by "`cluster`" image resource?

Reply:
Will add link to docu.